pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -873,7 +773,8 @@ stdenv.mkDerivation (finalAttrs: {
     interfaceVersion = 2;
 
     inherit withBootloader withCryptsetup withEfi withHostnamed withImportd withKmod
-      withLocaled withMachined withPortabled withTimedated withUtmp util-linux kmod kbd;
+      withLocaled withMachined withPortabled withTimedated withTpm2Tss withUtmp
+      util-linux kmod kbd;
```

Comment:
The styling from `nixfmt-rfc-style` is going to make this so much more readable.